public/secure/resize.js

```diff
@@ -0,0 +1,29 @@
+function handleResizeEvent(entries, elementId) {
+  // https://stackoverflow.com/a/58701523/5594539
+  window.requestAnimationFrame(() => {
+    if (!Array.isArray(entries) || !entries.length) return
+
+    const iframe = window.parent.document.getElementById(elementId)
+    iframe.style.height = `${window.document.body.offsetHeight}px`
```

Comment:
Really? You can do so? It's even better 👏

By the way, did you check if some iframe attribute could deny this?

Reply:
I think you can do so only if you are on the same origin.